### PR TITLE
Create microbial-cell.csl

### PR DESCRIPTION
**Pull request by s-hofer**

### CODE REVIEW

**Comment by rmzelle** [comment]
@s-hofer, is this for http://microbialcell.com/? (with reference guidelines at http://microbialcell.com/for-authors-2/manuscript-preparation/#specific_sections)

File: microbial-cell.csl [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    </names>
+  </macro>
+  <macro name="access">
+    <group>
+      <choose>
+        <if variable="DOI">
+          <text variable="DOI" form="short" prefix="doi: " suffix="."/>
+        </if>
+        <else>
+          <text variable="URL" prefix="doi:"/>
```
Comment (rmzelle) [inline microbial-cell.csl]: This DOI prefix is incorrect here. The URL variable generally won't hold a DOI URL.

File: microbial-cell.csl [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    </author>
+    <author>
+      <name>Frank Madeo</name>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="science"/>
+    <updated>2014-02-24T18:25:46+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+    <link/>
+    <link/>
```
Comment (rmzelle) [inline microbial-cell.csl]: Please make sure your style follows our [style requirements](https://github.com/citation-style-language/styles/wiki/Style-Requirements). In particular, the style needs to validate, and we need a "documentation" and a "template" link.

**Comment by s-hofer** [comment]
Dear Rintze,
yes indeed the faulty styles I uploaded were for microbialcell.com and 
cell-stress.com

Unfortunately I only uploaded and did not work on the styles myself. But 
I will look into the guidelines and re-upload a - hopefully - compatible 
version!

Thanks and have a great week,
Sebastian


Am 27.09.2017 um 06:08 schrieb Rintze M. Zelle:

**Comment by rmzelle** [comment]
> I will look into the guidelines and re-upload a - hopefully - compatible version!

Okay, great. I'll leave this open for now, then. You can update the existing pull request by following the instructions above (https://github.com/citation-style-language/styles/pull/2974#issuecomment-331650663).

**Comment by rmzelle** [comment]
@s-hofer, one of our most prolific contributors, @POBrien333, just submitted corrected CSL styles for both journals, so I'll close your pull requests. Don't hesitate to let us know if you run into any issues with these styles.

**Comment by s-hofer** [comment]
Thank you very much - you are the best!

And please forward my best regards to POBrien333.

Best wishes,
Sebastian

Am 11.10.2017 um 04:06 schrieb Rintze M. Zelle:

**Pull request closed without merging**